### PR TITLE
Add g:minimap_set_highlights to disable ColorScheme autocmd.

### PR DESCRIPTION
**Pull request by dsully**
## Check list

- [X] I have read through the [README](https://github.com/wfxr/minimap.vim/blob/master/README.md) (especially F.A.Q section)
- [X] I have searched through the existing issues or pull requests
- [X] I have verified all existing unit tests pass (See the README on how to run)
- [] I have added new tests if appropriate
- [X] I have performed a self-review of my code and commented hard-to-understand areas
- [X] I have made corresponding changes to the documentation (when necessary)

## Description

See issue #151 

## Type of change

- [ ] Bug fix
- [X] New feature
- [ ] Improvement of existing features
- [ ] Refactor
- [ ] Breaking change
- [ ] Documentation change
- [ ] CI / CD

## Test environment

- OS
    - [ ] Linux
    - [X] Mac OS X
    - [ ] Windows
    - [ ] Others:
- Vim
    - [X] Neovim: <Version>
    - [ ] Vim: <Version>

### CODE REVIEW

**Comment by ZNielsen** [comment]
Thanks for the PR! I'll take a look and let you know if anything needs tweaking.

File: README.md [hunk 1]
```diff
@@ -90,6 +90,8 @@ let g:minimap_auto_start_win_enter = 1
 | `g:minimap_highlight_range`                   | `0`                                                       | if set, minimap will highlight range of visible lines                |
 | `g:minimap_highlight_search`                  | `0`                                                       | if set, minimap will highlight searched patterns                     |
 | `g:minimap_git_colors`                        | `0`                                                       | if set, minimap will highlight range of changes as reported by git   |
+| `g:minimap_set_highlights`                    | `1`                                                       | if set, minimap will create an autocommand to set highlights on color scheme changes. |
```
Comment (ZNielsen) [inline README.md:93]: I think this name would benefit from being a bit more verbose. Some thing like `minimap_enable_highlight_colorgroup`.

File: plugin/minimap.vim [hunk 2]
```diff
@@ -103,21 +103,23 @@ highlight minimapRangeDiffLine     ctermbg=242 ctermfg=141 guibg=#4F4F4F guifg=#
...
-        \ highlight minimapDiffAdded                     ctermfg=148               guifg=#A4E400 |
-        \ highlight minimapDiffLine                      ctermfg=141               guifg=#AF87FF |
-        \ highlight minimapCursorDiffRemoved ctermbg=59  ctermfg=197 guibg=#5F5F5F guifg=#FC1A70 |
-        \ highlight minimapCursorDiffAdded   ctermbg=59  ctermfg=148 guibg=#5F5F5F guifg=#A4E400 |
-        \ highlight minimapCursorDiffLine    ctermbg=59  ctermfg=141 guibg=#5F5F5F guifg=#AF87FF |
-        \ highlight minimapRangeDiffRemoved  ctermbg=242 ctermfg=197 guibg=#4F4F4F guifg=#FC1A70 |
-        \ highlight minimapRangeDiffAdded    ctermbg=242 ctermfg=148 guibg=#4F4F4F guifg=#A4E400 |
-        \ highlight minimapRangeDiffLine     ctermbg=242 ctermfg=141 guibg=#4F4F4F guifg=#AF87FF
-augroup END
+if !exists('g:minimap_set_highlights')
```
Comment (ZNielsen) [inline plugin/minimap.vim:106]: We should explicitly set the property to 1 if the user didn't specify it so it can be queried later.

**Comment by ZNielsen** [comment]
Looks like the vim help in `doc/minimap-vim.txt` didn't get updated, can you add the new setting information in there as well?

**Comment by dsully** [comment]
> Looks like the vim help in `doc/minimap-vim.txt` didn't get updated, can you add the new setting information in there as well?

Will do. Some projects have that auto-generated from README.md, so I wasn't sure.

**Comment by ZNielsen** [comment]
> > Looks like the vim help in `doc/minimap-vim.txt` didn't get updated, can you add the new setting information in there as well?
> 
> Will do. Some projects have that auto-generated from README.md, so I wasn't sure.

Oh that would be nice. I'll look into that.

File: plugin/minimap.vim [hunk 3]
```diff
@@ -81,6 +81,10 @@ if !exists('g:minimap_git_colors')
     let g:minimap_git_colors = 0
 endif
 
+if !exists('g:minimap_enable_highlight_colorgroup')
+    let g:minimap_git_colors = 1
```
Comment (ZNielsen) [inline plugin/minimap.vim:85]: fix copy/paste

**Comment by ZNielsen** [comment]
Looks good! I just need to carve some time out to check out the branch and do some sanity checking hand-tests.

**Review by ZNielsen — APPROVED** [review]

**Comment by dsully** [comment]
What's left to get this merged? Thanks!

**Pull request merged**